react/shop-app/src/components/Footer/Footer.js

```diff
@@ -0,0 +1,127 @@
+import styled from "styled-components";
+import { BsFacebook } from 'react-icons/bs'
+import { AiFillInstagram } from 'react-icons/ai'
+import { BsTwitter } from 'react-icons/bs'
+
+import logo from '../../assets/logo.svg';
+
+
+const FooterContainer = styled.footer`
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+ 
+    img{
+        width:80px;
+        margin:20px 50px;
+
+    }
+    .social-media-logos{
+     
+    svg{
+        width:25px;
+        height:25px;
+        margin-right:10px;
+    }    
+    }
+    ul li{
+        list-style:none;
+        padding-bottom:5px
+    }
+    ul{
+        padding:0;
+    }
+
+    .footer-text{
+        border-top:1px solid white;
+        text-align:center;
+    }
+    .footer-container{
+        display:grid;
+        grid-template-columns:1fr;
+       
+    }
+    .footer-links{
+        display:flex;
+        justify-content:center;
+        align-items:center;
+
+        div{
+            margin-right:50px;
+        }
+    }
+    .footer-logos{
+        display:flex;
+        flex-direction:column;
+        justify-content:center;
+        align-items:center;
+    }
+    @media (min-width: 992px){
+        .footer-container{
+            display:grid;
+            grid-template-columns:auto 1fr;
+        }       
+        
+        .social-media-logos{
+            margin-left:20px;
+            margin-top:10px;
+    }
+    
+`
+
+function Footer() {
+    return (
+        <FooterContainer>
+            <div className="footer-container">
+            <div className="footer-logos">
+                <img src={logo} alt="logo"></img>
+                <div className="social-media-logos">
+                    <BsFacebook />
+                    <AiFillInstagram />
+                    <BsTwitter />
+                </div>
+            </div>
+            <div className="footer-links">
+                <div className="categories">
+                    <h3>Categories</h3>
+                    <ul>
```

Comment:
The hardcoded values in this list can be rendered dynamically from a constant. Check the NAV_ITEMS constant or the navigation component for the approach